orchagent/pfcactionhandler.h

```diff
@@ -39,15 +47,28 @@ class PfcWdActionHandler
         }
 
         static void initWdCounters(shared_ptr<Table> countersTable, const string &queueIdStr);
+        void initCounters(void);
+        void commitCounters(void);
+
+        virtual bool getHwCounters(PfcWdHwStats& counters)
```

Comment:
>counters [](start = 49, length = 8)

If base class has valid implementation for getHwCounters(), and you intend to return a zero counter, you should set it to zero explicitly.

Reply:
Done